daml/Marketplace/Distribution/Auction/Service.daml

```diff
@@ -39,9 +39,9 @@ template Service
           asset : Asset
           quotedAssetId : Id
           floorPrice : Decimal
-          assetDepositCid : ContractId AssetDeposit
+          depositCid : ContractId AssetDeposit
         do
-          create CreateAuctionRequest with ..
+          create CreateAuctionRequest with request = RequestCreateAuction with ..; ..
```

Comment:
This is not what I had in mind ... I was thinking

```
create CreateAuctionRequest with request = Auction
```

Comment continued below ...